openpype/hosts/maya/api/lib_renderproducts.py

```diff
@@ -80,6 +80,13 @@
 }
 
 
+def has_tokens(string, tokens):
+    """Return whether any of tokens is in input string (case-insensitive)"""
+    pattern = "({})".format("|".join(re.escape(token) for token in tokens))
```

Comment:
I believe this doesn't need to be grouped and the pattern can just be:
```suggestion
    pattern = "|".join(re.escape(token) for token in tokens)
```
```python
pattern = "|".join(re.escape(token) for token in tokens)
```

Reply:
Also, is this function unused? :)

Reply:
Hello Roy, thanks very much, I'm not entirely sure about this to be honest, will check it out later and let you know!

Reply:
Ah, seems like it's unused, will check with the guys perhaps it's of use later on.